doc/api/http2.md

```diff
@@ -128,7 +128,8 @@ solely on the API of the `Http2Session`.
 added: v8.4.0
 -->
 
-The `'close'` event is emitted once the `Http2Session` has been destroyed.
+The `'close'` event is emitted once the `Http2Session` has been destroyed. It
```

Comment:
Thank you!
Feel free to ignore, but maybe it would be a bit more clear if we write "Its listener does not expect any arguments"?

Reply:
Thanks! Good idea. I edited the wording to reflect the change.